src/main/resources/application-dev.yml

```
@@ -36,4 +36,44 @@ spring:
+
+---
+jwt:
+  secret: ${JWT_SECRET}
+  access-expiration: 2592000000 # 30일
+
+---
+spring:
+  web:
+    resources:
+      static-locations: classpath:/static/,classpath:/public/
+      add-mappings: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)